terraform/workspace_variables/review_aks.tfvars.json

```diff
@@ -0,0 +1,18 @@
+{
+  "paas_app_environment": "review",
+  "paas_cf_space": "bat-qa",
```

Comment:
if `deploy_aks` is true won't most of the `pass_*` variables be redundant?  `paas_app_environment` looks like it might be required but I don't think most of the rest are as those resources won't be deployed

Reply:
Variables are still required by terraform. We kept them to require minimal changes to the terraform code. Maybe add a future improvement to make them optional and remove them from tfvars?